source/manual/how-to-be-a-good-buddy.html.md

```diff
@@ -0,0 +1,40 @@
+---
+owner_slack: "#govuk-developers"
+title: How to be a good buddy
+section: Learning GOV.UK
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+When a new joiner starts on GOV.UK, it's good practice to assign them a "buddy" to help them get started on GOV.UK, explain it's infrastructure and provide initial support.
+
+## What is a buddy?
+
+A buddy acts as a temporary guide for new joiners. Their responsibilities include:
+
+- explaining how our stack works
+- pointing the new joiner to useful docs/slack channels/general resources
+- being on hand to answer specific questions
+- generally making them feel welcome
+
+Ideally, a buddy will be in the same team and sit within the same function as the new joiner eg: a new frontend developer would have a frontend buddy.
+
+## How does a buddy differ from a line manager?
+
+Line managers aren't the same as buddies in that they…
+
+- might not always be on the same team or even the same programme
+- are focused more on personal stuff like objectives and working patterns
+- handle general admin, HR processes and
```

Comment:
Should there be something after this "and"?

Reply:
That should've been "and SOP" but I'm now wondering if it's useful to include SOP in this. I'll amend.